docs/tutorial/dapp.md

```diff
@@ -2,46 +2,57 @@
 sidebar_position: 1
 ---
 
+import {NetworkButton} from '@site/src/components/NetworkButton';
+
 # Deploy and interact with your contract
 
-This guide will walk you through deploying and interacting with your contract in REMIX.
+This guide will walk you through deploying and interacting with contracts in REMIX.
 
 ## Deploy Your Contract
 
 1. Install MetaMask Wallet.
 
    Install the [MetaMask](https://support.metamask.io/getting-started/getting-started-with-metamask/#how-to-install-metamask) browser extension and create an Ethereum account. MetaMask will allow REMIX to interface with the Westend network.
-2. Launch REMIX.
 
-   Open [REMIX IDE](https://remix.polkadot.io) in your browser.
+2. Add Asset-Hub to your Metamask wallet, or switch to it if you have already added it.
 
-   Navigate to the **Deploy & Run** tab.
```

Comment:
Shouldn't this be done by REMIX when you switch the network there to deploy to Westend? Like it uses chainlist.org to do that for you?

Reply:
You mean it is supposed to? Because this change is not in our REMIX, yet?

Reply:
you have it here: https://github.com/paritytech/contract-docs/blob/master/docs/tutorial/dapp.md

Reply:
I guess REMIX would do the exact same thing. But maybe this button is better: When you open REMIX the correct network is already selected. But my plan was that REMIX always selects westend when you open because it doesn't support any other network.

Reply:
you can still use kittchensink node with remix